packages/iov-ethereum/src/ethereumconnection.ts

```diff
@@ -702,23 +826,106 @@ export class EthereumConnection implements BcpConnection {
         address: address,
         startblock: minHeight,
         endblock: maxHeight,
-        sort: "asc",
+        sort: "desc",
       },
     })).data;
-    if (responseBody.result === null) {
-      return [];
+    if (responseBody.result !== null) {
+      for (const tx of responseBody.result) {
+        if (tx.isError === "0" && tx.txreceipt_status === "1") {
+          // Do an extra query to the node as the scraper result does not contain the
+          // transaction signature, which we need for recovering the signer's pubkey.
+          const transaction = (await this.searchTransactionsById(Parse.transactionId(tx.hash)))[0];
+          out.push(transaction);
+        }
+      }
     }
+
+    return out;
+  }
+
+  private async searchSendTransactionsByAddressInLogs(
+    address: Address,
+    minHeight: number,
+    maxHeight: number,
```

Comment:
Could these have default values?

Reply:
Why would we need them? All callers set the values explicitly, which makes this function simpler.